### ISSUE TITLE
About training setting of real world BurstSR dataset

### ISSUE THREAD

**Issue by kmc3661** [akshaydudhane16/BIPNet#5]
Thank you for your wonderful work!

I'm trying to train with your code, and on the synthetic dataset, BipNet shows excellent results.
But on the real burst dataset, It does not give as good results as described in your paper.
The loss continues to increase except at the beginning.
your code set 1e-5 ~ 1e-6 with CosineAnnealingLR as a default setting.
If this learning rate is different from the setting of the experiment in the paper, could you share the training settings for the real burst dataset?

I'm really looking forward to your answer.

**Comment by RenLongWu** [comment]
Hi! How long time you train this model? Thank you!
